plugin/trino-postgresql/src/main/java/io/trino/plugin/postgresql/PostgreSqlConfig.java

```diff
@@ -55,4 +56,16 @@ public PostgreSqlConfig setIncludeSystemTables(boolean includeSystemTables)
         this.includeSystemTables = includeSystemTables;
         return this;
     }
+
+    public boolean isRemoteQueryCancellationEnabled()
+    {
+        return remoteQueryCancellationEnabled;
+    }
+
+    @Config("postgresql.remote-query-cancellation.enabled")
```

Comment:
It will be hard to migrate this to be a generic base-jdbc config... Why not do it right away?